python/app/dialog.py

```diff
@@ -9,7 +9,10 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 import sgtk
+from tank.util import sgre as re
```

Comment:
Can you get this from sgtk.util instead of tank.util? They're equivalent, but we're trying to not introduce new dependencies on the tank package.

Reply:
Fixed!